.github/workflows/build.yml

```diff
@@ -54,6 +54,18 @@ jobs:
       - name: Checkout Push to Registry action
         uses: actions/checkout@v4
 
+      # Confirm that cosign.pub matches SIGNING_SECRET
+      - uses: sigstore/cosign-installer@v3.2.0
+
+      - name: Check SIGNING_SECRET matches cosign.pub
```

Comment:
Shouldn't this use the same `if` condition as the other cosign steps, so that it doesn't cause failure when the keys are mismatched for a good reason?
`if: github.event_name != 'pull_request' && github.ref == 'refs/heads/live'`

Reply:
If we did that then a bad cosign.pub would only be caught after the pull request that caused the issue had landed.

Reply:
Ha, I see that the PR is failing here :facepalm: 

Is that because PRs don't have access to build secrets? If that's the case i'll add the `if` clause back in. This will mean that the issue is only caught when the live branch is built but that's better than nothing.

Reply:
> Is that because PR from untrusted contributors like me don't have access to build secrets? Is that because PR from untrusted contributors like me don't have access to build secrets? 

Yes, exactly.